auto_submit/lib/service/validation_service.dart

```diff
@@ -374,6 +374,11 @@ Exception: ${exception.message}
 
   /// A user can provide an alternate merge method via comment by providing the
   /// string @autosubmit:<merge_method>.
+  ///
+  /// This function will look for the first occurence of the regex in the latest
+  /// comment created in the pull request. If no comment is found with the
```

Comment:
"first occurence of the regex in the latest comment", is not accurate... -> "newest occurrence of the the regex in the comments"

Reply:
Yeah I am going to rework this.